### DISCUSSION TITLE
config file format

### DISCUSSION THREAD

**Discussion by araera111** [araera111/mysql-to-zod#25] [Ideas]
JSON ? YAML? TOML? 
js? cjs? mjs? ts?

**Comment by TonyGravagno** [comment]
I think the current `mysqlToZod.config.js` is fine and does not require change.
Since this project uses import rather than require, that implies modern ES, and therefore mjs rather than cjs.
I don't think it's required, but if mysqlToZod.config.js needs to import or use other ES functionality, rather than changing the extension to mjs, in package.json add `"type": "module",` to process .js files as ES modules.

JSON is more standard than other *ML, but with the options in code there is much more versatility - and since the code already uses the .js options, I don't see a need to change the format to something less capable like JSON.

HTH

**Comment by araera111** [comment]
Okay. Let's leave it at that for now.
Thanks for the helpful input.
